notebooks/multi_model_data/multi_model_data.ipynb

```diff
@@ -26,7 +26,7 @@
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": "! pip install -U ipyleaflet sqlalchemy-cratedb pandas<2.3"
+   "source": "! ! pip install -U -r https://github.com/crate/academy-fundamentals-course/raw/refs/heads/main/requirements.txt"
```

Comment:
`ipyleaflet` is not included in the `requirements.txt`. It is imported as `from ipyleaflet import Map, GeoJSON` in `multi_model_data.ipynb`. Does that part still work?